server/utils/placementRequests/mapTaskForSummaryList.ts

```diff
@@ -0,0 +1,20 @@
+import { ApprovedPremisesApplication as Application, PlacementApplication } from '../../@types/shared'
```

Comment:
This is a new function, so the commit message probably doesn't reflect what's going on here